<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iqgui</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gui.qrc">
    <normaloff>:/icons/icon.png</normaloff>:/icons/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Navigation</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Starting frame:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>No. of frames:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBox_sframes">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1500</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBox_nframes">
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>200</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLCDNumber" name="lcdNumber_sframes">
           <property name="digitCount">
            <number>7</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Method:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLCDNumber" name="lcdNumber_thld_max">
           <property name="smallDecimalPoint">
            <bool>false</bool>
           </property>
           <property name="digitCount">
            <number>7</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
           <property name="value" stdset="0">
            <double>1000000.000000000000000</double>
           </property>
           <property name="intValue" stdset="0">
            <number>1000000</number>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QCheckBox" name="checkBox_mask">
           <property name="text">
            <string>Mask min.</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Contrast max.:</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QComboBox" name="comboBox_window"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Starting time [s]:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Points per frames:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QCheckBox" name="checkBox_log">
           <property name="text">
            <string>Log</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QComboBox" name="comboBox_method"/>
         </item>
         <item row="13" column="1">
          <widget class="QComboBox" name="comboBox_color"/>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_lframes">
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>1024</number>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Window:</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Color map:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QCheckBox" name="checkBox_info">
           <property name="text">
            <string>Show Info</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLCDNumber" name="lcdNumber_thld_min">
           <property name="digitCount">
            <number>7</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
           <property name="value" stdset="0">
            <double>0.000000000000000</double>
           </property>
           <property name="intValue" stdset="0">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Contrast min.:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Actions</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="3" column="1">
          <widget class="QPushButton" name="pushButton_save_csv">
           <property name="text">
            <string>Save CSV</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_load_conf">
           <property name="text">
            <string>Load Config</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_choose_file">
           <property name="toolTip">
            <string>Open file [cmd] or [ct</string>
           </property>
           <property name="text">
            <string>Choose File</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_save_conf">
           <property name="text">
            <string>Save Config</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_replot">
           <property name="toolTip">
            <string>Plot [return]</string>
           </property>
           <property name="text">
            <string>Plot</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>File Info</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTextBrowser" name="textBrowser"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Sliders</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSlider" name="verticalSlider_sframes">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="verticalSlider_thld_min">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>1000000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="pageStep">
          <number>1000</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="verticalSlider_thld_max">
         <property name="maximum">
          <number>1000000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="pageStep">
          <number>1000</number>
         </property>
         <property name="value">
          <number>1000000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Plot</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="MatplotLibWidget" name="mplWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionChoose_file"/>
    <addaction name="actionReplot"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReplot">
   <property name="text">
    <string>Replot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionChoose_file">
   <property name="text">
    <string>Choose file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatplotLibWidget</class>
   <extends>QWidget</extends>
   <header>matplotlibwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinBox_lframes</tabstop>
  <tabstop>spinBox_nframes</tabstop>
  <tabstop>spinBox_sframes</tabstop>
  <tabstop>comboBox_color</tabstop>
  <tabstop>verticalSlider_sframes</tabstop>
  <tabstop>verticalSlider_thld_min</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>spinBox_sframes</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_sframes</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_sframes</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox_sframes</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_thld_min</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_thld_min</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_thld_max</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_thld_max</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
